### PR TITLE
added: Alaya&PlatON testnet

### PR DESCRIPTION
**Pull request by benbaley**
added: Alaya&PlatON testnet

### CODE REVIEW

File: _data/chains/eip155-201030.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  "infoURL": "https://www.alaya.network/",
+  "shortName": "alaya",
+  "chainId": 201030,
+  "networkId": 1,
+  "icon": "alaya",
+  "explorers": [
+    {
+      "name": "alaya explorer",
+      "url": "devnetscan.alaya.network",
```
Comment (ligi) [inline _data/chains/eip155-201030.json:25]: the explorer URL should start with the protocol (e.g. https://)

File: _data/chains/eip155-210309.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  "infoURL": "https://www.platon.network",
+  "shortName": "PlatON",
+  "chainId": 210309,
+  "networkId": 1,
+  "icon": "platon",
+  "explorers": [
+    {
+      "name": "PlatON explorer",
+      "url": "devnetscan.platon.network",
```
Comment (ligi) [inline _data/chains/eip155-210309.json:25]: the explorer URL should start with the protocol (e.g. https://)

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-210309.json (hunk 2 above)
Comment (benbaley) [reply _data/chains/eip155-210309.json]: fixed
please take a look again, thank you very much!

File: _data/chains/eip155-201030.json (hunk 1 above)
Comment (benbaley) [reply _data/chains/eip155-201030.json]: fixed
please take a look again, thank you very much!

**Comment by biganxin** [comment]
Excuse me  @ligi  I have two questions for you.
1, We have configured the Token faucet for the above network, but the 【buy】 button on the page is not clickable?
2, We have configured the logo, but it is not showing up?
How can we fix these problems? look forward to your help, thank you!
image

**Comment by ligi** [comment]
you need to contact MetaMask for these problems - not us

**Comment by biganxin** [comment]
OKey ,Thank u

**Pull request merged**